plugins/woocommerce/tests/api-core-tests/ci-release.global-setup.js

```diff
@@ -7,6 +7,8 @@ const zipPath = path.resolve( 'tmp', 'woocommerce.zip' );
 const downloadURL = `https://github.com/woocommerce/woocommerce/releases/download/${ UPDATE_WC }/woocommerce.zip`;
 
 test( `Setup remote test site`, async ( { page, request } ) => {
+	test.setTimeout( 5 * 60 * 1000 );
```

Comment:
We need to apply a generous amount of timeout because downloading the WooCommerce zip and uploading it to the release test site take quite some time to complete.